src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Setup/MissingSetupComponentRegistrationService.cs

```diff
@@ -45,10 +45,10 @@ internal class MissingSetupComponentRegistrationService : OnceInitializedOnceDis
     // State
     private readonly ConcurrentHashSet<string> _webComponentIdsDetected = new(StringComparers.VisualStudioSetupComponentIds);
     private readonly ConcurrentHashSet<string> _missingRuntimesRegistered = new(StringComparers.WorkloadNames);
-    private readonly ConcurrentDictionary<Guid, ConcurrentHashSet<WorkloadDescriptor>> _projectGuidToWorkloadDescriptorsMap = new();
```

Comment:
Note: I'll say I prefer this form because it puts the components--"Guid" and "WorkloadDescriptors"--in the same order as the type parameters.

Reply:
I see the logic. My preference is more about the usage. For example:

```c#
var user = _userByName[name];

_userByName[name] = user;
```

There's no objectively better approach. If you like we can discuss this as a team and come up with a standard approach throughout.